A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1998,12 +1998,18 @@ class FMCMainDisplay extends BaseAirliners {
 
     setPerfApprVApp(s) {
         if (s === FMCMainDisplay.clrValue) {
-            this.vApp = NaN;
-        }
-        const value = parseFloat(s);
-        if (isFinite(value) && value > 0) {
-            this.vApp = value;
-            return true;
+            if (isFinite(this.vApp)) {
+                this.vApp = NaN;
+                return true;
+            }
+        } else {
+            const value = parseFloat(s);
+            if (isFinite(value) && value > 0) {
```

Comment:
The Range is 90 to VMO, so it should be >=90 and <=350.

Reply:
Done :)